src/main/java/dk/cngroup/lentils/controller/ClientController.java

```diff
@@ -0,0 +1,41 @@
+package dk.cngroup.lentils.controller;
+
+import dk.cngroup.lentils.entity.Team;
+import dk.cngroup.lentils.service.*;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.*;
+
+
+@Controller
+@RequestMapping("/client")
+public class ClientController {
+
+    private CypherService cypherService;
+    private TeamService teamService;
+    private StatusService statusService;
+    private HintService hintService;
+
+    @Autowired
+    public ClientController(CypherService cypherService,
+                            TeamService teamService,
+                            StatusService statusService,
+                            HintService hintService) {
+        this.cypherService = cypherService;
+        this.teamService = teamService;
+        this.statusService = statusService;
+        this.hintService = hintService;
+    }
+
+
+    //temporary team(will be team logged in), cypherstatus
+    @GetMapping(value = "/list")
+    public String listAllCyphers(Model model){
+        model.addAttribute("cyphers", cypherService.getAll());
+        model.addAttribute("team", new Team("Developeri",4));
```

Comment:
Tady by mel být tým načtený z DB, ne ?

Reply:
Přesně tak. Akorát na základě přihlášeného uživatele/týmu.
Neměl by to být parametr v URL. Jinak by šlo teoreticky jednoduše listovat datama ostatních týmu.

Ale přihlašování jsme ještě nějak nedořešili. Možná by jsme s tím mohli pohnout příští týden.

Reply:
Dal jsem prozatím jako issue a dořešíme to v týdnu, jak bude čas.

Use case s tímhle jako Hotový neznačím, takže na to doufám nezapomenu.

Reply:
Souhlasim, zatim neresit.